com.unity.render-pipelines.universal/Shaders/Utils/FallbackLoading.shader

```diff
@@ -0,0 +1,91 @@
+Shader "Hidden/Universal Render Pipeline/FallbackLoading"
+{
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "RenderPipeline" = "UniversalPipeline" "IgnoreProjector" = "True" "ShaderModel" = "4.5"}
+
+        Pass
+        {
+            HLSLPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            #pragma target 4.5
+            #pragma multi_compile _ UNITY_SINGLE_PASS_STEREO STEREO_INSTANCING_ON STEREO_MULTIVIEW_ON
+            #pragma multi_compile _ DOTS_INSTANCING_ON
+            #pragma editor_sync_compilation
+
+            #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl"
+
+            struct appdata_t
+            {
+                float4 vertex : POSITION;
+                UNITY_VERTEX_INPUT_INSTANCE_ID
+            };
+
+            struct v2f
+            {
+                float4 vertex : SV_POSITION;
+                UNITY_VERTEX_OUTPUT_STEREO
+            };
+
+            v2f vert (appdata_t v)
+            {
+                v2f o;
+                UNITY_SETUP_INSTANCE_ID(v);
+                UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(o);
+                o.vertex = TransformObjectToHClip(v.vertex.xyz);
+                return o;
+            }
+
+            float4 frag (v2f i) : SV_Target
+            {
+                return float4(0,1,1,1);
+            }
+            ENDHLSL
+        }
+    }
+
+    SubShader
+    {
+        Pass
+        {
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            #pragma target 2.0
+            #pragma multi_compile _ UNITY_SINGLE_PASS_STEREO STEREO_INSTANCING_ON STEREO_MULTIVIEW_ON
+            #pragma editor_sync_compilation
+            #include "UnityCG.cginc"
```

Comment:
I don't understand why this sub shader is for Builtin not URP. We should have a fallback to display the loading error in URP for platforms that dont support SM 4.5.

Reply:
This sub shader is mostly here for consistency and isn't actually strictly needed. I removed it in the latest push as agreed on Slack